content/creator/sdk7/2d-ui/ui-positioning.md

```diff
@@ -29,44 +30,156 @@ ReactEcsRenderer.setUiRenderer(() => (
+> TIP: Decentraland's UI implementation is based on that of [Yoga](https://yogalayout.com/docs/). Read [this article](https://www.joshwcomeau.com/css/interactive-guide-to-flexbox/) for a very approachable and in-depth coverage of the properties available in Flexbox.
+
+### Arranging child entities
 
 By default, child entities are positioned in relation to the top-left corner of its parent. You can use properties like `justifyContent` and `alignItems` to change this behavior.
 
-> Tip: When measuring from the top, the numbers for `position` should be negative. Example: to position a component leaving a margin of 20 pixels with respect to the parent on the top and left sides, set `position` to 20, -20.
+> Tip: Any properties that refer to _content_ refer to entities along the main axis (determined by `flexDirection`). Any properties that refer 
+
+- `flexDirection`: Flex direction controls the direction in which children of a node are laid out. This is also referred to as the main axis. The main axis is the direction in which children are laid out. The cross axis is the axis perpendicular to the main axis, or the axis which wrapping lines are laid out in. It takes its value from the `YGFlexDirection` enum. The following options are available:
+
+	- `YGFlexDirection.YGFD_ROW` (DEFAULT)
+	- `YGFlexDirection.YGFD_ROW_REVERSE`
+	- `YGFlexDirection.YGFD_COLUMN` 
+	- `YGFlexDirection.YGFD_COLUMN_REVERSE`
+
+
+- `justifyContent`: This property describes how to align children within the main axis of their container. For example, you can use this property to center a child horizontally within a container with `flexDirection` set to row or vertically within a container with `flexDirection` set to column. The value of this property must be from the `YGJustify` enum. Possible values are:
+
+	- `YGAlign.FLEX_START` (DEFAULT): Align children of a container to the start of the container's main axis.
+
+	- `YGAlign.FLEX_END`: Align children of a container to the end of the container's main axis.
+
+	- `YGAlign.CENTER`: Align children of a container in the center of the container's main axis.
+
+	- `YGAlign.SPACE_BETWEEN`: Evenly space of children across the container's main axis, distributing remaining space between the children.
+
+	- `YGAlign.SPACE_AROUND`: Evenly space of children across the container's main axis, distributing remaining space around the children. Compared to space between using space around will result in space being distributed to the beginning of the first child and end of the last child.
+
+	- `YGAlign.SPACE_EVENLY`: Evenly distributed within the alignment container along the main axis. The spacing between each pair of adjacent items, the main-start edge and the first item, and the main-end edge and the last item, are all exactly the same.
```

Comment:
When enumerating things, preventing the empty line will not add a paragraph separation between the elements.
```suggestion
	- `YGAlign.FLEX_START` (DEFAULT): Align children of a container to the start of the container's main axis.
	- `YGAlign.FLEX_END`: Align children of a container to the end of the container's main axis.
	- `YGAlign.CENTER`: Align children of a container in the center of the container's main axis.
	- `YGAlign.SPACE_BETWEEN`: Evenly space of children across the container's main axis, distributing remaining space between the children.
	- `YGAlign.SPACE_AROUND`: Evenly space of children across the container's main axis, distributing remaining space around the children. Compared to space between using space around will result in space being distributed to the beginning of the first child and end of the last child.
	- `YGAlign.SPACE_EVENLY`: Evenly distributed within the alignment container along the main axis. The spacing between each pair of adjacent items, the main-start edge and the first item, and the main-end edge and the last item, are all exactly the same.
```